### PR TITLE
Add link to verify commit into slack reminder message

### PR DESCRIPTION
**Pull request by praveenperera**
#2

### CODE REVIEW

File: app/controllers/commits_controller.rb [hunk 1]
```diff
@@ -12,8 +12,16 @@ def index
     prepare_commits
   end
 
+  def show
```
Comment (vipulnsward) [inline app/controllers/commits_controller.rb]: We should instead just redirect from here to commits list home page for now, the checks below will be take care of there.

**Comment by praveenperera** [comment]
Yes that makes since there shouldn't be too many unverified commits, making specific commit easy to find. I'll make the change.

**Comment by praveenperera** [comment]
@vipulnsward  Need a clarification are you saying the link in the message should link to `/commit/2/` and have the action redirect to `/` or have the message link directly to list home page?

File: app/views/commits/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div class="to-verify-commits">
+  <%= render "shared/commit_list", commits: [@commit] %>
```
Comment (slavingia) [inline app/views/commits/show.html.erb]: do we need this html file anymore?

**Pull request merged**